.github/workflows/install-zcashd.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  workflow_call:
+
+jobs:
+  install-zcashd:
```

Comment:
Unification case: targets for zcash are called `install-zcashd` which in fact does building also and for zebra there is a target `build-zebra` which does the same (so building and uploading artifacts). 
Both should be unified (but maybe that's the plan for zebra) 😄

Reply:
Done